src/esp/assets/CollisionMeshData.h

```diff
@@ -14,11 +18,15 @@
 
 namespace esp {
 namespace assets {
-
-//! Reference to vertices and
-//! Usage: (1) for creating collision mesh in Bullet
+/**
+ * @brief Provides references to geometry and topology for an individual
+ * component of an asset for use in generating collision shapes for simulation.
+ * Usage: (1) for creating collision mesh/convex in @ref
+ * physics::BulletPhysicsManager and @ref physics::BulletRigidObject
+ */
 struct CollisionMeshData {
-  //! Primitive type (has to be triangle for Bullet to work)
+  //! Primitive type (has to be triangle for Bullet to work). See @ref
+  //! BulletRigidObject::constructBulletCompoundFromMeshes.
```

Comment:
(Apologies if I'm not up-to-date on coding conventions, but) why `//!` here and `/**` everywhere else?

Reply:
No good reason, should be consistent. 👍